### PR TITLE
Run Jasmine specs via Github action.

### PR DESCRIPTION
**Pull request by giulliano-bueno**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "scripts": {
-    "test": "jasmine-browser-runner runSpecs --config=src/test/javascripts/jasmine-browser.json"
+    "test": "jasmine-browser-runner runSpecs"
```
Comment (jacekbilski) [inline package.json:3]: I'm against this change. It's mainly Java project, and in those tests are usually kept in `src/test/`.

File: spec/support/jasmine-browser.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
+
```
Comment (jacekbilski) [inline spec/support/jasmine-browser.json:1]: You're adding this file for a second time. Please review your PR after merging from original master branch.

File: README.md [hunk 3]
```diff
@@ -50,12 +50,19 @@ Hints:
...
+docker run --name backend-tester --rm "backend:tester"
+```
+
+### Jasmine
 
-Under Linux you _might_ also need to set `OPENSSL_CONF=/dev/null` environment variable.
-Without that PhantomJS might run into trouble.
-Also, make sure you have `chromedriver` on your path.
-Those two problems will be solved once we Dockerize test environment.
+The frontend code is tested using a Javascript test framework called Jasmine. The package.json file defines its configuration.
```
Comment (jacekbilski) [inline README.md:65]: Probably "too fast" merged changes from master. Those lines are important, without those I, and maybe others, will not be able to execute the tests locally.

**Review by jacekbilski — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (giulliano-bueno) [reply README.md:65]: Are you able to run it today?

Comment (jacekbilski) [reply README.md:65]: Yes, but only when I do exactly those two things mentioned in README. So:
`PATH="/snap/chromium/current/usr/lib/chromium-browser":"$PATH" OPENSSL_CONF=/dev/null ./gradlew clean test`

**Pull request closed without merging**